react-native/index.d.ts

```diff
@@ -2563,8 +2563,18 @@ declare module "react" {
         ref?: Ref<DrawerLayoutAndroidStatic & ViewStatic>
     }
 
+    interface DrawerPosition {
```

Comment:
I'm relatively new to typescript as well, but I made an almost identical change to @victor95pc but it seems to have disappeared (maybe during the transition to the @types branch?..):

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11358

If you look at the RN example for DrawerLayoutAndroid they also access the DrawerConsts via the static DrawerLayoutAndroid.positions.Left/Right accessors.

https://facebook.github.io/react-native/docs/drawerlayoutandroid.html

I also made the following commit changing the drawerPosition property from 'any' to 'number' since DrawerLayoutAndroid.positions.Left = DrawerConsts.DrawerPosition.Left = Gravity.START =  8388611 (https://developer.android.com/reference/android/view/Gravity.html) and a similar definition for the corresponding right position but that seems to have disappeared as well.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/13118